src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -0,0 +1,494 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
Also add `// eslint-disable-next-line import/no-extraneous-dependencies` here to suppress the GitHub Actions warning.